src/functions-and-arrays.js

```diff
@@ -78,7 +190,26 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(arr,word) {
+  if (arr.length === 0) {
+    return 0;
+  }
+
+  let count = 0;
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] === word) {
+      count++;
+    }
+  }
+
+  if (count === 0) {
+    return 0;
+  } else if (count === 1) {
+    return 1;
+  } else {
+    return 5;
+  }
+}
```

Comment:
why is it returning 5 in the else statement? is enough just returning the value stored in count

Reply:
That ask Jasmine . I was wondering and i try to deleted now ,but give the last part red.   
![Screenshot from 2023-03-29 21-26-58](https://user-images.githubusercontent.com/126404329/228647000-6d5939cd-4036-4886-9eb0-7b69ec99e4bf.png)

Reply:
so i think that needs the else {return 5}